src/me/corriekay/pokegoutil/windows/PokemonTab.java

```diff
@@ -706,6 +766,7 @@ private PokemonTableModel(PokemonGo go, List<Pokemon> pokes, PokemonTable pt) {
                 dustToLevelCol.add(i.getValue(), p.getStardustCostsForPowerup());
                 pokeballCol.add(i.getValue(), WordUtils.capitalize(p.getPokeball().toString().toLowerCase().replaceAll("item_", "").replaceAll("_", " ")));
                 caughtCol.add(i.getValue(), LocalDateTime.ofInstant(Instant.ofEpochMilli(p.getCreationTimeMs()), ZoneId.systemDefault()));
+                favCol.add(i.getValue(), p.isFavorite());
```

Comment:
Oh, nearly forgot. I don't know how that bool is displayed in the column, but wouldn't it be cooler to just display something if it is a favorite?

So do `(p.isFavorite()) ? "True" : ""` here.
That makes the column even better sortable and it's better to see what are favorites. With the new UI, we will add yellow stars :P